src/cwts/networkanalysis/run/RunNetworkLayout.java

```diff
@@ -0,0 +1,454 @@
+                {
+                    try
+                    {
+                        if ((argIndex + 1) >= args.length)
+                            throw new NumberFormatException();
+                        requiredNQualityValueImprovements = Integer.parseInt(args[argIndex + 1]);
+                        if (requiredNQualityValueImprovements <= 0)
+                            throw new NumberFormatException();
+                    }
+                    catch (NumberFormatException e)
+                    {
+                        throw new IllegalArgumentException("Value must be a positive integer number.");
+                    }
+                    argIndex += 2;
+                }
+                else if (arg.equals("--seed"))
+                {
+                    try
+                    {
+                        if ((argIndex + 1) >= args.length)
+                            throw new NumberFormatException();
+                        seed = Long.parseLong(args[argIndex + 1]);
+                    }
+                    catch (NumberFormatException e)
+                    {
+                        throw new IllegalArgumentException("Value must be an integer number.");
+                    }
+                    useSeed = true;
+                    argIndex += 2;
+                }
+                else if (arg.equals("-w") || arg.equals("--weighted-edges"))
+                {
+                    weightedEdges = true;
+                    argIndex++;
+                }
+                else if (arg.equals("--sorted-edge-list"))
+                {
+                    sortedEdgeList = true;
+                    argIndex++;
+                }
+                else if (arg.equals("--input-layout"))
+                {
+                    if ((argIndex + 1) >= args.length)
+                        throw new IllegalArgumentException("Missing value.");
+                    initialLayoutFilename = args[argIndex + 1];
+                    argIndex += 2;
+                }
+                else if (arg.equals("-o") || arg.equals("--output-layout"))
+                {
+                    if ((argIndex + 1) >= args.length)
+                        throw new IllegalArgumentException("Missing value.");
+                    finalLayoutFilename = args[argIndex + 1];
+                    argIndex += 2;
+                }
+                else
+                    throw new IllegalArgumentException("Invalid command line argument.");
+            }
+            catch (IllegalArgumentException e)
+            {
+                System.err.print("Error while processing command line argument " + arg + ": " + e.getMessage() + "\n\n" + USAGE);
+                System.exit(-1);
+            }
+        }
+        if (argIndex >= args.length)
+        {
+            System.err.print("Error while processing command line arguments: Incorrect number of command line arguments.\n\n" + USAGE);
+            System.exit(-1);
```

Comment:
You could consider continuing with trying to parse all other arguments, in order to raise all relevant exceptions, and then exit afterwards. Right now, people will only receive on error message, even though there might be multiple errors.